src/zrc/datamodel/models/betrokkene.py

```diff
@@ -0,0 +1,212 @@
+import logging
+
+from django.contrib.postgres.fields import ArrayField
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from vng_api_common.fields import BSNField, RSINField
+
+from ..constants import GeslachtsAanduiding, SoortRechtsvorm
+from .base_models import Rol, ZaakObject
+from .zaakobjecten import ZakelijkRechtHeeftAlsGerechtigde
+
+logger = logging.getLogger(__name__)
+
+
+class RolZaakobjectRelationMixin(models.Model):
```

Comment:
```suggestion
class AbstractRolZaakobjectRelation(models.Model):
```

It looks weird to see classes inherit from just a "mixin". This is of course not a mixin (mixins should only add behaviour via methods and possibly some attributes), but an abstract base model.

Reply:
abstract model classes are renamed